mitodl/ocw-hugo-projects
# course image & thumbnail

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-projects/issues/23

#### What's this PR do?
This PR adds 2 `RelationField` widgets to the `ocw-course` starter config for both Course Image and Course Image Thumbnail, allowing the images to be uploaded as separate resources and then linked to in the config.

#### How should this be manually tested?
 - Convert `ocw-course/ocw-studio.yaml` to JSON and paste it into a `WebsiteStarter` config in `ocw-studio`
 - Create a `Website` from the `WebsiteStarter` you just created
 - Upload 2 separate image resources into the Resources collection
 - Go to Metadata under Settings and select your 2 separate images in the Course Image and Course Image Thumbnail sections
 - Verify that the Metadata saves properly
